### PR TITLE
Comtac cm2

### PR DESCRIPTION
**Pull request by J2thatsme**
Bit library is newer on master. Test will work after merge

### CODE REVIEW

File: types/comtac/CM-1/uplink.spec.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+before(function (done) {
+  fs.readFile(
+    __dirname + "/default.schema.json",
+    "utf8",
+    function (err, fileContents) {
+      if (err) throw err;
+      defaultSchema = JSON.parse(fileContents);
+    }
```
Comment (mariusgiger) [inline types/comtac/CM-1/uplink.spec.js:26]: split into two befores

File: types/comtac/CM-2/uplink.spec.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+before(function (done) {
+  fs.readFile(
+    __dirname + "/default.schema.json",
+    "utf8",
+    function (err, fileContents) {
+      if (err) throw err;
+      defaultSchema = JSON.parse(fileContents);
+    }
+  );
+  fs.readFile(
```
Comment (mariusgiger) [inline types/comtac/CM-2/uplink.spec.js:28]: split into two befores

**Review by mariusgiger — CHANGES_REQUESTED** [review]
please fix the tests

File: types/comtac/CM-1/uplink.spec.js (hunk 1 above)
Comment (J2thatsme) [reply types/comtac/CM-1/uplink.spec.js:26]: test works with the new addition in the Bits library

**Pull request merged**